backend/node-register/apps.py

```diff
@@ -1,10 +1,31 @@
+import time
+import logging
 from django.apps import AppConfig
+from django.conf import settings
 from substrapp.ledger_utils import invoke_ledger
 
 
+logger = logging.getLogger(__name__)
+
+
 class NodeRegisterConfig(AppConfig):
     name = 'node-register'
 
     def ready(self):
-        # args is set to empty because fabric-sdk-py doesn't allow None args for invoke operations
-        invoke_ledger(fcn='registerNode', args=[''], sync=True)
+
+        # We try until success if dev mode, if it fails the backend will not start
+        # In production we let this app fails
+        while True:
+            try:
+                # args is set to empty because fabric-sdk-py doesn't allow None args for invoke operations
+                invoke_ledger(fcn='registerNode', args=[''], sync=True)
+            except Exception as e:
```

Comment:
Could we catch a specific exception? Like a `TimeoutError` or a `GrpcError` for instance (the traceback is quite hard to follow).

Reply:
Yes, I will put `RpcError` in the except catch.